packages/tpetra/core/src/Tpetra_Details_makeOptimizedColMap.hpp

```diff
@@ -378,24 +367,19 @@ namespace Details {
   /// It does everything that that function does, except that it does
   /// not compute a new Import.
   template<class MapType>
-  MapType
+  Teuchos::RCP<const MapType>
```

Comment:
It' doesn't really matter whether we return Map by const or nonconst RCP.  We could get rid of const here.  We may want to take a vote on that.